applications/luci-app-watchcat/htdocs/luci-static/resources/view/watchcat.js

```diff
@@ -0,0 +1,101 @@
+		o.value('ping_reboot', _('Ping Reboot'));
+		o.value('periodic_reboot', _('Periodic Reboot'));
+		o.value('restart_iface', _('Restart Interface'));
+
+		o = s.taboption('general', form.Value, 'period', 
+			_('Period'), 
+			_("In Periodic Reboot mode, it defines how often to reboot. <br> \
+				In Ping Reboot mode, it defines the longest period of \
+				time without a reply from the Host To Check before a reboot is engaged. <br> \
+				In Network Restart mode, it defines the longest period of \
+				time without a reply from the Host to Check before the interface is restarted. \
+				<br><br>The default unit is seconds, without a suffix, but you can use the \
+				suffix <b>m</b> for minutes, <b>h</b> for hours or <b>d</b> \
+				for days. <br><br>Examples:<ul><li>10 seconds would be: <b>10</b> or <b>10s</b></li><li>5 minutes would be: <b>5m</b></li><li> \
+				1 hour would be: <b>1h</b></li><li>1 week would be: <b>7d</b></li><ul>"));
+		o.default = '6h';
+
+		o = s.taboption('general', form.Value, 'pinghosts', _('Host To Check'), _(`IPv4 address or hostname to ping.`));
+		o.datatype = 'host(1)';
+		o.default = '8.8.8.8';
+		o.depends({ mode: "ping_reboot" });
+		o.depends({ mode: "restart_iface" });
+
+		o = s.taboption('general', form.Value, 'pingperiod', 
+			_('Check Interval'), 
+			_("How often to ping the host specified above. \
+				<br><br>The default unit is seconds, without a suffix, but you can use the suffix <b>m</b> for minutes, <b>h</b> for hours or <b>d</b> for days. <br><br> \
+				Examples:<ul><li>10 seconds would be: <b>10</b> or <b>10s</b></li><li>5 minutes would be: <b>5m</b></li><li>1 hour would be: <b>1h</b></li><li>1 week would be: <b>7d</b></li><ul>"));
+		o.default = '30s';
+		o.depends({ mode: "ping_reboot" });
+		o.depends({ mode: "restart_iface" });
+
+		o = s.taboption('general', form.ListValue, 'pingsize', 
+			_('Ping Packet Size'));
+		o.value('small', _('Small: 1 byte'));
+		o.value('windows', _('Windows: 32 bytes'));
+		o.value('standard', _('Standard: 56 bytes'));
+		o.value('big', _('Big: 248 bytes'));
+		o.value('huge', _('Huge: 1492 bytes'));
+		o.value('jumbo', _('Jumbo: 9000 bytes'));
+		o.default = 'standard';
+		o.depends({ mode: 'ping_reboot' });
+		o.depends({ mode: 'restart_iface' });
+
+		o = s.taboption('general', form.Value, 'forcedelay',
+			_('Force Reboot Delay'),
+			_("Applies to Ping Reboot and Periodic Reboot modes</i> <br> When rebooting the router, the service will trigger a soft reboot. \
+				Entering a non-zero value here will trigger a delayed hard reboot if the soft reboot were to fail. \
+				Enter the number of seconds to wait for the soft reboot to fail or use 0 to disable the forced reboot delay."));
+		o.default = '1m';
+		o.depends({ mode: 'ping_reboot' });
+		o.depends({ mode: 'periodic_reboot' });
+
+		o = s.taboption('general', widgets.NetworkSelect, 'interface',
```

Comment:
This should be:
`		o = s.taboption('general', widgets.DeviceSelect, 'interface',
`

Reply:
This is a cherrypick from the master so we should update this as well.